app/tests/container_manager/test_container_manager.py

```
@@ -13,19 +13,89 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import importlib
 from asyncio import TimeoutError
```

Ruff (0.16.3)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)